src/mono/netcore/System.Private.CoreLib/src/LinkerDescriptor/System.Private.CoreLib.xml

```diff
@@ -817,5 +817,9 @@
 		<type fullname="Interop/Globalization">
 			<method name="GetICUVersion" />
 		</type>
+		<type fullname="System.Runtime.InteropServices.Marshal">
+			<!-- marshal-ilgen.c: emit_marshal_custom_get_instance -->
+			<method name="GetCustomMarshalerInstance" />
+		</type>
```

Comment:
@marek-safar Is this generally enough for the linker?  This worked locally but on CI so I figure the new internal class + new field are preserved, but the test still seems to fail on CI

Reply:
It's not ideal but this should be enough.